pyInterface/compareAmplitudeFiles.py

```diff
@@ -43,7 +43,7 @@ def saveTree(path, name, title, lst):
 	for element in lst:
 		arr[0] = element
 		tree.Fill()
-	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
+# 	hist = ROOT.TH1D(name + "_hist", title, 301, -1e-11, 1e-11)
```

Comment:
Setting the binning directly in the Draw call is how the ROOT reference guide recommends to solve this problem. 
saveTree() will not be called in this file anyway.